### ISSUE TITLE
About Page

### ISSUE THREAD

**Issue by Alonski** [Alonski/CoinPanion#1]
Page explaining how CoinPanion works

**Comment by Josephrp** [comment]
I suggest instead to take this opportunity to make the about page very "human", have pictures of us, the team, and a few funny sentences (2) each description - hopefully warm and funny?

**Comment by Alonski** [comment]
@Josephrp Good idea.
Once we start you can draft up a `human` about page that looks nice?

**Comment by Josephrp** [comment]
yes sure of course - you know i'm doing my best... I'll make sure i lie about everyone to make you folks seem human.

It's so crazy it just might work!

**Comment by Josephrp** [comment]
Propose to separate the "Team Page" from the "About Page"

adding this here as a kickoff for About page: 

WE ALL HAVE SPECIAL PEOPLE THAT MERIT OUR ATTENTION. THESE PEOPLE ARE OUR GRANDMAS, COLLEGUES, NEIHBORS, OR COINPANIONS.

REGULARLY PAY YOUR COINPANIONS. CREDIT CONTRACTS BETWEEN YOU AND THOSE THAT DESERVE IT.

WE USE THE ETH PUBLIC CHAIN TO HELP YOU REACH MILLIONS OF POTENTIAL BENEFICIARIES WORLDWIDE. STATUS.IM GIVES A MOBILE HOME TO YOUR BANK ACCOUNT.

SIMPLY, CONNECT, FIND, SEND, AND ENJOY!

**Comment by rhlsthrm** [comment]
We should also have some text in there that says you can support people who create things you love. Right now it seems only for people you personally know.

**Comment by Josephrp** [comment]
All will be improved and re written :-)

**Comment by Josephrp** [comment]
Current version of About page is here: https://drive.google.com/open?id=0B8C3EFaoqIvddzI2ZzI1MXJDaE0
